docs/ManualDeploymentGuideScalarDBServerOnEKS.md

```diff
@@ -0,0 +1,199 @@
+# Deploy Scalar DB Server on AWS
+
+The Scalar DB server is a standalone gRPC server that implements the Scalar DB interface.
+We can deploy the Scalar DB server on any Kubernetes service.
+This guide will help you to deploy a Scalar DB server in AWS EKS.
+
+## What we create
+
+![image](images/scalardbserver-eks-diagram.png)
+
+In this guide, we will create the following components.
+
+* A VPC with NAT gateway
+* An EKS cluster with two Kubernetes node groups
+* A managed database service supported by Scalar DB.
+* A Bastion instance with a public IP
+* Amazon CloudWatch
+
+
+## Step 1. Configure your network
+
+Configure a secure network according to your organizational standard or application requirements. This section shows how to configure a secure network for Scalar DB server deployment.
+
+### Requirements
+
+* You must create VPC with public NAT gateways on private networks. NAT gateway is necessary to enable internet access for Kubernetes node group subnets.
+  * Internet access is required to pull Scalar DB server image for Kubernetes deployment.
+* You must follow this [AWS official guide](https://docs.aws.amazon.com/eks/latest/userguide/network_reqs.html) when creating subnets for EKS.
+
+### Recommendations
+
+* You should create private subnets for the Kubernetes cluster for production.
+* You should create a bastion server to manage the Kubernetes cluster.
+* You should create 3 subnets in 3 availability zones for the Kubernetes cluster for higher availability.
+* You should create subnets with the prefix at least `/24` for the Kubernetes cluster to work without issues even after scaling.
+
+### Steps
+
+* Create an Amazon VPC based on the [AWS official guide](https://docs.aws.amazon.com/eks/latest/userguide/creating-a-vpc.html) with the above requirements and recommendations.
+
+## Step 2. Set up a database
+
+In this section, you will set up a database for the Scalar DB server.
+
+### Requirements
+
+* You must have a database that the Scalar DB server supports.
+
+### Steps
+
+* Follow [Set up a database guide](./SetupDatabase.md) to set up a database for Scalar DB.
+
+## Step 3. Configure EKS
+
+This section shows how to create an EKS cluster and 2 managed node groups (one for Scalar DB server and envoy and one for logs and metrics collection) for the Scalar DB server and monitor agent deployment.
```

Comment:
```suggestion
This section shows how to create an EKS cluster and 2 managed node groups (one for Scalar DB server and envoy, and the other for logs and metrics collection) for the Scalar DB server and monitor agent deployment.
```

Reply:
Thank you.
I have updated the sentence as suggested.